bundles/org.eclipse.swt/Eclipse SWT PI/win32/org/eclipse/swt/internal/win32/OS.java

```diff
@@ -497,6 +497,7 @@ public class OS extends C {
 	public static final int FR_PRIVATE = 0x10;
 	public static final int FSHIFT = 0x4;
 	public static final int FVIRTKEY = 0x1;
+	public static final int GA_ROOT = 0x2;
```

Comment:
From what I have seen in the `OS` class so far, not all predefined argument / flag values are provided together with a method accepting them, but they are added on-demand. Maybe this was by accident and actually it does no harm to define all predefined argument values together with a method. So it is of course okay to do so, but I would not expect it to be mandatory, as it would only add constants that are unused.